src/BernardPublisher.php

```diff
@@ -15,7 +15,7 @@ class BernardPublisher implements Publisher
     private $queueResolver;
     private $type;
 
-    public function __construct(MessageInEnvelopSerializer $serializer, Producer $bernard, RoutingKeyResolver $queueResolver, $type)
+    public function __construct(MessageInEnvelopeSerializer $serializer, Producer $bernard, RoutingKeyResolver $queueResolver, $type)
```

Comment:
This will be a BC break, right?

One may say that a bundles BC promise only include a service's public methods. Im not sure how this is defined in the SimpleBus project. Here is an example: http://docs.php-http.org/en/latest/httplug/backwards-compatibility.html#symfony-bundle